docs/installation.rst

```diff
@@ -198,10 +174,10 @@ To start the dev server, run ``./manage.py runserver``, then open up
 ``http://localhost:8000``. If everything's working, you should see
 the MDN home page!
 
-You might need to first set ``LC_CTYPE`` if you're on Mac OS X until
-`bug 754728 <https://bugzilla.mozilla.org/show_bug.cgi?id=754728>`_ is fixed::
+You might need to first set ``LC_CTYPE`` in your ``.env`` file if you're on
+Mac OS X until `bug 754728 <https://bugzil.la/754728>`_ is fixed::
```

Comment:
Ok, I looked into this a bit more, this originates here: http://babel.edgewall.org/ticket/200 and was fixed in Babel trunk here: http://babel.edgewall.org/changeset/586

And we're indeed still vulnerable to that problem: https://github.com/mozilla/kuma/blob/827fbdfc77773021b8f2fdcd452d73d91f913e6e/vendor/packages/Babel/babel/core.py#L642

So as long as we use a stone age old version of Babel we're stuck on this section.